docs/CHANGELOG.rst

```diff
@@ -9,10 +9,20 @@ This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Unreleased
 ==========
 
+Added
+-----
+- Add support for ``markduplicates`` reporting in ``multiqc`` and
+  implement it in ``wgbs-single`` and ``wgbs-paired`` workflow
+- Add ``insert-size`` process as a part of QC steps in ``wgbs-paired``
+  workflow
+
 Changed
 -------
 - **BACKWARD INCOMPATIBLE:** Require Resolwe 21.x
 - Use human samples for ``alignment-star`` testing
+- Move wgbs test files to its own folders
+- **BACKWARD INCOMPATIBLE:** Split ``wgbs`` workflow into 
```

Comment:
Backward incompatible changes should be listed at the top of the changelog entry,